.github/ISSUE_TEMPLATE/Standard-Feature-Issue-Template.md

```diff
@@ -22,31 +22,45 @@ so that < why do you need this (short description of the business value >.
 ## Notes
 < bulleted list of special notes or suggestions to the development team on the implementation of the changes. >  
 
-## Definition of Ready
+## Ready for Planning
 
 - [ ] The Issue maintains standard format for Issues and meets INVEST
-- [ ] Issue Defined and written appropriately
-- [ ] Acceptance Criteria Defined
-- [ ] User acceptance test cases defined
+- [ ] Story and Business Value complete
+- [ ] Initial Acceptance Criteria Defined
 - [ ] The person who will Approve story identified
-- [ ] Dependencies Identified
+
+## Ready for Ready to Work
+- [ ] Acceptance Criteria Refined
 - [ ] Issue size estimated by Dev Team
-- [ ] Proposed solution accepted
+- [ ] Dependencies Identified
 
-## Definition of Done
+## Ready for Review
+- [ ] Proposed solution defined and accepted by Approver
+- [ ] Code implemented and built
+- [ ] Testing implemented and passed
 
-- [ ] Issue accepted by PO
-- [ ] Issue is scored
+## Ready for Pre-Production QA
+- [ ] Peer Reviews complete with all changes tested
 - [ ] All features implemented
 - [ ] All features not yet implemented or inactive are hidden from the user
-- [ ] Peer Reviews complete	
-- [ ] Unit and regression tests are Passed
-- [ ] Manual Testing Complete and issues resolved
-- [ ] Automated tests created and run successfully
-- [ ] All blockers addressed
+- [ ] Special testing instructions noted
+- [ ] Acceptance Test cases created and reviewed by QA
```

Comment:
I'm thinking the three amigos (QA - PO - DEV) should be involved in this review, maybe after a first internal review from QA team only. This is to everybody knows what is being tested because any of the amigos might be uncovering certain scenarios that haven't been thought initially, some inconsistent logic, etc. This means static quality assurance that will save time, effort, and avoid things going to prod that are not easy to spot in the code by regular testing methods.